plugins/stack-overflow/config.d.ts

```diff
@@ -24,5 +24,11 @@ export interface Config {
      * @visibility frontend
      */
     baseUrl?: string;
+
+    /**
+     * The api key to authenticate to Stack Overflow API
+     * @visibility secret
+     */
+    apiKey?: string;
```

Comment:
Just to confirm: is this change necessary anymore, @mmulligan03?  Wondering if it can just be removed entirely.